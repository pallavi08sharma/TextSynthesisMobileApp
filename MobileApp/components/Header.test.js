import React from 'react';
import renderer from 'react-test-renderer';
import Header from '../components/Header';


const mockedNavigate = jest.fn();
jest.mock('@react-navigation/native', () => (
  { useNavigation: () => ({ navigate: mockedNavigate }) }));

describe('<Header />', () => {
  it('has 2 child', async() => {
    const tree = renderer.create(<Header />).toJSON();
    expect(tree.children.length).toBe(2);
  });
  it('renders correctly', async() => {
    const tree = renderer.create(<Header />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});