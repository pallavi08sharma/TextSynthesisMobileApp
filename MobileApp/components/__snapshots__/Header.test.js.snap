// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#fa7da7",
      "display": "flex",
      "elevation": 5,
      "flexDirection": "row",
      "height": 50,
      "justifyContent": "space-between",
      "left": 0,
      "paddingHorizontal": 20,
      "position": "absolute",
      "top": 30,
      "width": "100%",
    }
  }
>
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
  >
    <Text />
  </View>
  <View>
    <Text />
  </View>
</View>
`;
